test/functional/test_runner.py

```diff
@@ -766,7 +766,7 @@ def get_next(self):
                         self.jobs.remove(job)
                         print(f"{name} failed at attempt {attempt}/{self.attempts}, Duration: {int(time.time() - start_time)} s")
                         # start over
-                        portseed_arg = ["--portseed={}".format(portseed)]
+                        portseed_arg = ["--portseed={}".format(portseed + (1000 * attempt))]
```

Comment:
0574b44ae741fd35dbd9fab0b1e4f4d45bd40846: seems unrelated, should be done in a separate PR

Reply:
Dropped 0574b44ae741fd35dbd9fab0b1e4f4d45bd40846 from PR, resolved in latest push.